docs/root/intro/arch_overview/http/http3.rst

```diff
@@ -0,0 +1,52 @@
+.. _arch_overview_http3:
+
+HTTP3 overview
+==============
+
+.. warning::
+
+  HTTP/3 support is still in Alpha, and should be used with caution.
+  Outstanding issues required for HTTP/3 to go GA can be found
+  `here <https://github.com/envoyproxy/envoy/labels/quic-mvp>`_
+  For example QUIC does not currently support in-place filter chain updates, so users
+  requiring dynamic config reload for QUIC should wait until
+  `#13115 <https://github.com/envoyproxy/envoy/issues/13115>`_ has been addressed.
+
+  For general feature requests beyond production readiness, you can track
+  the `area-quic <https://github.com/envoyproxy/envoy/labels/area%2Fquic>`_ tag.
+
+HTTP3 downstream
+----------------
+
+Downstream Envoy HTTP/3 support can be turned up via adding
+:ref:`quic_options <envoy_v3_api_field_config.listener.v3.UdpListenerConfig.quic_options>` and
+ensuring the downstream transport socket is a QuicDownstreamTransport.
```

Comment:
The HCM config needs to specify codec type to be `Http3` as well.

Reply:
Yeah, at the point you configure a listener, the rest should be auto-validated also shown by the sample config.
happy to add the detail in a follow-up but I think i'm going to land this version today :-)